tests/webdriver-e2e/page-objects/common.page.js

```diff
@@ -0,0 +1,18 @@
+class CommonPage {
+  get hamburgerMenu() { return $('#header-dropdown-link'); }
+  get logoutButton() { return $('.fa-power-off'); }
+  get modalBody() { return $('div.modal-body'); }
+  get yesButton() { return $('a.btn.submit.btn-danger'); }
+  // const modal = $('div.modal-dialog');
+  async logout() {
+    await (await this.hamburgerMenu).click();
+    await (await this.logoutButton).click();
+    (await this.modalBody).waitForDisplayed();
+    const warning = await (await this.modalBody).getText();
+    await (await this.yesButton).click();
+    // await helper.waitUntilReadyNative(element(by.css('form#form')));
+    return warning;
```

Comment:
I think this is a confusing thing to return, when the function name says "logout". 

I think we should three-step this, like: 

1. one function that clicks the logout button
2. another function that gets the modal text 
3. another function that validates the modal

Based on: https://en.wikipedia.org/wiki/Single-responsibility_principle

Reply:
Agreed, I was just migrating the existing function. I'll update before submitting a proper PR